go/kbcmf/packets.go

```diff
@@ -6,36 +6,40 @@ package kbcmf
 import ()
 
 type receiverKeysPlaintext struct {
+	_struct    bool   `codec:",toarray"`
```

Comment:
Neat feature of the msgpack library.  Made this change trivial.

On Thursday, December 3, 2015, oconnor663 <notifications@github.com> wrote:

> In go/kbcmf/packets.go
> <https://github.com/keybase/client/pull/1459#discussion_r46515125>:
>
> > @@ -6,36 +6,40 @@ package kbcmf
> >  import ()
> >
> >  type receiverKeysPlaintext struct {
> > +	_struct    bool   `codec:",toarray"`
>
> What is this wizardry?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1459/files#r46515125>.
>